### PR TITLE
Remove reduntant @Values from parameter documentation

### PR DESCRIPTION
**Pull request by peterbarker**
we have enforced this one-of-Values-or-Bitmask in libraries for a long time, and should have been enforcing it in vehicles before now.

MissionPlanner and MAVProxy ignore Values if Bitmask is present

Built on top of https://github.com/ArduPilot/ardupilot/pull/27070

### CODE REVIEW

File: ArduPlane/Parameters.cpp [hunk 1]
```diff
@@ -1135,8 +1132,7 @@ const AP_Param::GroupInfo ParametersG2::var_info[] = {
     // @Param: DSPOILER_OPTS
     // @DisplayName: Differential spoiler and crow flaps options
     // @Description: Differential spoiler and crow flaps options
-    // @Values: 0: none, 1: D spoilers have pitch input, 2: use both control surfaces on each wing for roll, 4: Progressive crow flaps only first (0-50% flap in) then crow flaps (50 - 100% flap in)
```
Comment (amilcarlucas) [inline ArduPlane/Parameters.cpp:1138]: Some of the details should be added in the description IMHO

**Review by amilcarlucas — CHANGES_REQUESTED** [review]
Nice PR!!!

File: Rover/Parameters.cpp [hunk 2]
```diff
@@ -636,8 +635,7 @@ const AP_Param::GroupInfo ParametersG2::var_info[] = {
     // @Param: FS_OPTIONS
     // @DisplayName: Failsafe Options
     // @Description: Bitmask to enable failsafe options
-    // @Values: 0:None,1:Failsafe enabled in Hold mode
-    // @Bitmask: 0:Failsafe enabled in Hold mode
+    // @Bitmask: 0:Failsafe enabled in Hold mode,1:Failsafe enabled in Hold mode
```
[the remark this answers is not in the document]
Comment (peterbarker) [reply Rover/Parameters.cpp]: Fixed.

File: ArduPlane/Parameters.cpp (hunk 1 above)
Comment (peterbarker) [reply ArduPlane/Parameters.cpp:1138]: Done

**Pull request merged**